packages/cascara/src/system-components/Boundaries/fallbacks/ErrorFallback.js

```diff
@@ -8,18 +9,44 @@ const propTypes = {
 };
 
 const ErrorFallback = ({ error, resetErrorBoundary }) => {
+  const checkedFromLS =
+    window.localStorage.getItem('ESPRESSIVE_SHOW_ERRORS') === 'true';
+  const [checked, setChecked] = useState(checkedFromLS);
+  const showError = process.env.NODE_ENV === 'development' || checked;
+
+  const hangleCheckboxClick = useCallback(() => {
+    setChecked(!checked);
+  }, [checked]);
+
+  const hangleTryAgainClick = useCallback(() => {
+    window.localStorage.setItem('ESPRESSIVE_SHOW_ERRORS', checked);
+
+    if (resetErrorBoundary) {
+      resetErrorBoundary();
+    }
+  }, [checked, resetErrorBoundary]);
+
   return (
-    <div className='ui tiny error message' role='alert'>
+    <Role className='ui tiny error message' role='alert'>
       <div className='header'>Something went wrong:</div>
-      <pre>{error?.message}</pre>
+      {showError && <pre>{error?.message}</pre>}
+      <br />
       <button
```

Comment:
@lordpixel should be using Button from reakit instead?

Reply:
@lordpixel Yes, that would be good, but I think we could also just swap this over officially for the Button component from Cascara instead... which is using Reakit. We could also remove the classname values and use the props on that Button instead. If the negative type styles are not there yet, let's still use that type and fix the styles as a separate task.